training.md

```diff
@@ -124,6 +124,7 @@ This section is divided into sort filters, sort of. See each section and find ou
  - https://blog.heroku.com/python37-dataclasses-async-await - well written summary of what's new in python-3.7;
  - https://realpython.com/python38-new-features/ - cool new features in Python 3.8;
  - https://realpython.com/python39-new-features/ - cool new features in Python 3.9;
+ - https://www.dabeaz.com/python/UnderstandingGIL.pdf - GIL War
```

Comment:
should it be for all python devs? It kind of seems like a senior-only concern, you wouldn't expect a junior or intermediate to have to meddle with this